docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/00 - Introduction to study parameters.md

```diff
@@ -0,0 +1,96 @@
+  Configuration Module.  Older projects should ensure there is no
+  overlap in settings between this file and the ConfigSettings
+  table.
+
+
+#### Troubleshooting Tips
+
+If Loris is having trouble with critical path settings impacting
+the front-end (e.g. finding the main codebase or the css), it
+may be difficult to use the front-end Configuration module to
+correct these path settings.  These path variables can also be
+accessed and updated via the back-end, where they are stored in
+the Config database table.
+
+- To view all configuration settings from the back-end, the following
+  query can be run in the MySQL command line:
+
+```sql
+     SELECT c.ConfigID, cs.Name, cs.Label, c.Value, cs.Description
+        FROM Config c
+        LEFT JOIN ConfigSettings cs ON (cs.ID=c.ConfigID);
+```
+
+> For setup troubleshooting, ConfigSettings under the `Paths` and
+  `WWW` sections are important
+
+
+- To view **path** settings (subset of configuration settings),
+  the following query can be run in the MySQL command line:
+
+```sql
+     SELECT c.ConfigID, cs.Name, cs.Label, c.Value, cs.Description
+        FROM Config c LEFT JOIN ConfigSettings cs ON (c.ConfigID = cs.ID)
+            JOIN ConfigSettings csp ON (cs.Parent = csp.ID)
+        WHERE csp.Name='paths';
+```
+
+- To view all **www** settings (subset of configuration settings),
+  using the following query:
+
+```sql
+     SELECT c.ConfigID, cs.Name, cs.Label, c.Value, cs.Description
+         FROM Config c LEFT JOIN ConfigSettings cs ON (c.ConfigID = cs.ID)
+             JOIN ConfigSettings csp ON (cs.Parent = csp.ID)
+     WHERE csp.Name='www'; ```
+
+- You may also need to change your URL and HOST settings, which
+  you can do with these commands:
+
+```sql
+UPDATE Config SET Value='$yourURL'
+     WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='url');
+UPDATE Config SET Value='$yourHostName'
+     WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='host');
+```
+
+- ConfigSetting **url** value should not terminate in a slash.
+  e.g. "http(s)://localhost" not "http(s)://localhost/". To fix,
+  run:
+
+```sql
+UPDATE Config SET Value='http(s)://localhost'
+     WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='host');
+```
+
+- If you get an error after clicking "Submit" or "Save data" on a
+  form, check that ***url*** Config setting is set for your host.
+  (Previous iterations of LORIS recommended setting this to the empty
+  string)  Run:
+
+```sql
+UPDATE Config SET Value='$yourhost'
+     WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='url');
+```
+
+- If your dashboard loads but no other modules load, ensure that
+  your `/var/apache2/apache2.conf` file is set to `AllowOverride
+  All` in the section `<Directory /var/www/>` to enable re-write
+  rules (based on `htdocs/.htaccess`)
+
+- Ensure your `smarty/templates_c` directory is Apache-writable
+  by running `composer update`. This will also update your dependencies.
```

Comment:
Huh? Why would `composer update` have anything to do with `smarty/templates_c`? I think those are 2 separate and unrelated points.

Reply:
yeah actually this should probably done with `make all` anyway.

Reply:
I don't think either `make all` or `composer update` do anything about the templates_c directory.

Reply:
Yes, you're right -- sorry I was making a tangential point above.

Reply:
I think they were two points that got accidentally put on one line